instaloader/instaloader.py

```diff
@@ -204,6 +206,22 @@ def __init__(self,
         self.resume_prefix = resume_prefix
         self.check_resume_bbd = check_resume_bbd
 
+        self.slide = slide or ""
+        self.slide_start = 0
+        self.slide_end = 0
+        if self.slide != "":
+            splitted = self.slide.split('-')
+
+            if len(splitted) == 1:
+                self.slide_start = self.slide_end = int(splitted[0])
+                if self.slide_start <= 0:
+                    raise InvalidArgumentException("--slide parameter cannot be negative")
+            elif len(splitted) == 2 and int(splitted[0]) < int(splitted[1]) and int(splitted[0]) > 0:
+                self.slide_start = int(splitted[0])
+                self.slide_end = int(splitted[1])
+            else:
+                raise InvalidArgumentException("Invalid data for --slide parameter: ")
```

Comment:
This prints the message `Invalid data for --slide parameter: ` without that anything follows the colon. Either add the `self.slide` input to the message, or replace the colon by a period.

Reply:
Yeah, my bad.